docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md

```diff
@@ -78,7 +78,23 @@ Installing YugabyteDB Anywhere on airgapped hosts, without access to any Interne
   `https://downloads.yugabyte.com`
   `https://download.docker.com`
 
-- Ensuring that Docker Engine version 20.10.n is available. If it is not installed, you need to follow the procedure described in [Installing Docker in airgapped](https://www.replicated.com/docs/kb/supporting-your-customers/installing-docker-in-airgapped/).
+- Ensuring that Docker Engine version 20.10.n is available. If it is not installed, you need to follow the procedure described in [Installing Docker in airgapped](https://community.replicated.com/t/installing-docker-in-airgapped-environments/81).
+
+    If you want to set up Docker on Amazon Linux OS, perform the following:
+
+    1. Install Docker using the following commands:
+
+        ```sh
+        sudo yum install docker
+        sudo systemctl daemon-reload
+        ```
+
+    1. Start Docker using the following command:
+
+        ```sh
+        sudo systemctl start docker.service
+        ```
+
 - Ensuring that the following ports are open on the YugabyteDB Anywhere host:
```

Comment:
> Are the rest of these bullets really only for airgapped? except for the airgapped package, they look like they apply to any install. why is storage here 100GB and 200GB above?

@iSignal , Can you please check this page for its accuracy as there has been some revamp to the Prerequisite section recently. So we want to make sure that the details on this page are valid. Here's the deploy preview for the page : https://deploy-preview-16629--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/default/

Reply:
1. The bullet point for `Ensuring that the attached disk storage (such as persistent EBS volumes on AWS) is 100 GB minimum.` in this section - https://deploy-preview-16629--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/default/#airgapped-hosts -  could be removed because we state prereqs = 200 GB upfront.

Other than that, the prereqs page looks good.